apps/aefate/src/aefa_fate_op.erl

```diff
@@ -1095,7 +1095,7 @@ make_fate_tx(Aetx, GAMetas, PayFor, ES) ->
              make_some(Res)}
     end.
 
-make_fate_base_tx(BaseTxType, BaseTx) ->
+make_fate_base_tx(BaseTxType, BaseTx, ES) ->
```

Comment:
For what?

Reply:
Yes, seems to not be needed. (Note make_fate_tx is available only from FATE_2)

Reply:
It is used now, missed an argument to abort in the first version.